Tests/helper.py

```diff
@@ -19,42 +20,40 @@
 
 logger = logging.getLogger(__name__)
 
-
-HAS_UPLOADER = False
-
+uploader = None
 if os.environ.get("SHOW_ERRORS"):
-    # local img.show for errors.
-    HAS_UPLOADER = True
-
-    class test_image_results:
-        @staticmethod
-        def upload(a, b):
-            a.show()
-            b.show()
-
+    uploader = "show"
 elif "GITHUB_ACTIONS" in os.environ:
-    HAS_UPLOADER = True
-
-    class test_image_results:
-        @staticmethod
-        def upload(a, b):
-            dir_errors = os.path.join(os.path.dirname(__file__), "errors")
-            os.makedirs(dir_errors, exist_ok=True)
-            tmpdir = tempfile.mkdtemp(dir=dir_errors)
-            a.save(os.path.join(tmpdir, "a.png"))
-            b.save(os.path.join(tmpdir, "b.png"))
-            return tmpdir
-
+    uploader = "github_actions"
 else:
     try:
         import test_image_results
 
-        HAS_UPLOADER = True
+        uploader = "aws"
     except ImportError:
         pass
 
 
-def convert_to_comparable(a, b):
+def upload(a: Image.Image, b: Image.Image) -> str | None:
+    if uploader == "show":
+        # local img.show for errors.
+        a.show()
+        b.show()
+    elif uploader == "github_actions":
+        dir_errors = os.path.join(os.path.dirname(__file__), "errors")
+        os.makedirs(dir_errors, exist_ok=True)
+        tmpdir = tempfile.mkdtemp(dir=dir_errors)
+        a.save(os.path.join(tmpdir, "a.png"))
+        b.save(os.path.join(tmpdir, "b.png"))
+        return tmpdir
+    elif uploader == "aws":
+        return test_image_results.upload(a, b)
```

Comment:
Out of scope for this PR, but if the AWS Lambda is long gone, shall we remove this stuff?

https://github.com/wiredfool/test-image-results/pull/1#issuecomment-1898821181

Reply:
Ok, I've created https://github.com/python-pillow/Pillow/pull/7739 to remove it.